lib/src/third_party/boringssl/ffigen.yaml

```diff
@@ -0,0 +1,264 @@
+name: BoringSsl
+description: 'Bindings to BoringSSL.'
+output: 'lib/src/third_party/boringssl/generated_bindings.dart'
+headers:
+  entry-points:
+    - 'third_party/boringssl/src/include/openssl/*.h'
+compiler-opts: '-Ithird_party/boringssl/src/include -I/Library/Developer/CommandLineTools/usr/lib/clang/12.0.0/include'
+comments:
+  style: any
+  length: full
+sort: true
+typedef-map:
+  # TODO(dacoharkes): https://github.com/dart-lang/sdk/issues/42563 https://github.com/dart-lang/ffigen/issues/7
+  size_t: IntPtr
+macros:
+  include:
+    - AES_BLOCK_SIZE
+    - EC_PKEY_NO_PUBKEY
+    - EVP_PKEY_EC
+    - EVP_PKEY_RSA
+    - HKDF_R_OUTPUT_TOO_LARGE
+    - NID_secp384r1
+    - NID_secp521r1
+    - NID_X9_62_prime256v1
+    - RSA_PKCS1_OAEP_PADDING
+    - RSA_PKCS1_PADDING
+    - RSA_PKCS1_PSS_PADDING
+enums:
+  include:
+    - point_conversion_form_t
+unnamed-enums:
+  include:
+    - ERR_LIB_HKDF
+structs:
+  include:
+    - 'nothing'
+  rename:
+    bignum_ctx: BN_CTX
+    bignum_st: BIGNUM
+    bn_blinding_st: BN_BLINDING
+    bn_gencb_st: BN_GENCB
+    bn_mont_ctx_st: BN_MONT_CTX
+    cbb_st: CBB
+    cbs_st: CBS
+    crypto_ex_data_st: CRYPTO_EX_DATA
+    ecdsa_sig_st: ECDSA_SIG
+    ec_group_st: EC_GROUP
+    ec_key_st: EC_KEY
+    ec_point_st: EC_POINT
+    engine_st: ENGINE
+    env_md_ctx_st: EVP_MD_CTX
+    env_md_st: EVP_MD
+    evp_aead_st: EVP_AEAD
+    evp_cipher_ctx_st: EVP_CIPHER_CTX
+    evp_cipher_st: EVP_CIPHER
+    evp_pkey_asn1_method_st: EVP_PKEY_ASN1_METHOD
+    evp_pkey_ctx_st: EVP_PKEY_CTX
+    evp_pkey_st: EVP_PKEY
+    hmac_ctx_st: HMAC_CTX
+    rsa_meth_st: RSA_METHOD
+    rsa_st: RSA
+  dependency-only: opaque
+functions:
+  include:
+  - BN_bin2bn
```

Comment:
This duplicates: https://github.com/google/webcrypto.dart/blob/32ea48b8c4581814f8f751eb2a4ab9bda5551a78/src/symbols.yaml#L45

We should at a minimum put a comment in both places, that they must be kept in sync...

Another option is that make the generator that reads `symbols.yaml` use contents of this file and the other ffigen config file..